src/registrar/context_processors.py

```diff
@@ -121,8 +120,47 @@ def is_widescreen_mode(request):
         hasattr(request.user, "is_org_user")
         and request.user.is_org_user(request)
         and any(path in request.path for path in portfolio_widescreen_paths)
-        and not any(exclude_path in request.path for exclude_path in exclude_paths)
     )
 
+    is_excluded = any(exclude_path in request.path for exclude_path in exclude_paths)
+
+    # Return a dictionary with the widescreen mode status.
+    return {
+        "is_widescreen_mode": (is_widescreen or is_portfolio_widescreen or get_is_widescreen_left_justified(request))
+        and not is_excluded
+    }
+
+
+def get_is_widescreen_left_justified(request):
```

Comment:
Is this really needed? Seems like this is the pattern we want as default.

Reply:
(..slack thread..)